google/cloud/aiplatform/datasets/dataset.py

```diff
@@ -77,45 +155,37 @@ def create(
         metadata_schema_uri: str,
         gcs_source: Optional[Sequence[str]] = None,
         bq_source: Optional[str] = None,
+        labels: Optional[Dict] = {},
```

Comment:
The benefit of having this might not outweigh the confusion it would cause.

Reply:
The alternative would be to just support `Sequence[str]`. What does everyone think?

Reply:
Oh yes, `labels` is named poorly. Maybe in MBSDK we can rename it as metadata_labels?

Reply:
created a ticket for resource_labels to resources